app/sagas/login.js

```diff
@@ -229,6 +239,7 @@ const handleLoginSuccess = function* handleLoginSuccess({ user }) {
 		const server = yield select(getServer);
 		yield put(encryptionInit());
 		yield put(roomsRequest());
+		yield fork(fetchAppActionsFork);
```

Comment:
This fetch is "soft blocking" `getPermissions` now, so there's a race condition.
Soft blocking because it's only preventing from settings the permissions on redux and not from fetching from server.
`fork` from redux-saga makes all these fetch to work in parallel, so although they're in sequence, it's not really behaving like that.

To fix it without overcomplicating, I think we can move the logic from `fetchAppActionsFork` to `fetchPermissionsFork`, before `yield getPermissions();`.